### PR TITLE
[SYCLomatic] Add mapping query for 10 dnn APIs

### PR DESCRIPTION
**Pull request by intwanghao**

### CODE REVIEW

**Review by tangjj11 — APPROVED** [review]
LGTM

File: clang/test/dpct/query_api_mapping/cuDNN/dnn_part1.cu [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+// cudnnActivationBackward-NEXT: Is migrated to:
+// cudnnActivationBackward-NEXT:     h.async_activation_backward(desc, *alpha, dst_d, dst, diff_dst_d, diff_dst, src_d, src, *beta, diff_src_d, diff_src);
+// cudnnActivationBackward-EMPTY:
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=cudnnActivationForward | FileCheck %s -check-prefix=cudnnActivationForward
+// cudnnActivationForward: CUDA API:
+// cudnnActivationForward-NEXT: cudnnActivationForward(
+// cudnnActivationForward-NEXT:     h /*cudnnHandle_t*/, desc /*cudnnActivationDescriptor_t*/,
+// cudnnActivationForward-NEXT:     alpha /*void **/, src_d /*cudnnTensorDescriptor_t*/, src /*void **/,
+// cudnnActivationForward-NEXT:     beta /*void **/, dst_d /*cudnnTensorDescriptor_t*/, dst /*void **/);
```
Comment (zhimingwang36) [inline clang/test/dpct/query_api_mapping/cuDNN/dnn_part1.cu:33]: here in migrated code,  h is not clear:  1.  type info in sycl side.  2. where is from, sycl  or dpct::. 

how about we add:  cudnnHandle_t  h in "//begin and //end",  then h will be more clear.

File: clang/test/dpct/query_api_mapping/cuDNN/dnn_part1.cu [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+// cudnnSetTensor4dDescriptor-EMPTY:
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=cudnnGetTensor4dDescriptor | FileCheck %s -check-prefix=cudnnGetTensor4dDescriptor
+// cudnnGetTensor4dDescriptor: CUDA API:
+// cudnnGetTensor4dDescriptor-NEXT: cudnnGetTensor4dDescriptor(d /*cudnnTensorDescriptor_t*/,
+// cudnnGetTensor4dDescriptor-NEXT:     t /*cudnnDataType_t **/, n /*int **/, c /*int **/,
+// cudnnGetTensor4dDescriptor-NEXT:     h /*int **/, w /*int **/, ns /*int **/,
+// cudnnGetTensor4dDescriptor-NEXT:     cs /*int **/, hs /*int **/, ws /*int **/);
+// cudnnGetTensor4dDescriptor-NEXT: Is migrated to:
+// cudnnGetTensor4dDescriptor-NEXT:       d.get(t, n, c, h, w, ns, cs, hs, ws);
```
Comment (zhimingwang36) [inline clang/test/dpct/query_api_mapping/cuDNN/dnn_part1.cu]: same as above.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**